tests/cases/providers_care_platform/case_providers_participants.yaml

```diff
@@ -0,0 +1,166 @@
+#CXTB-170: Participant Page Navigation
+TestAccessParticipantDirectoryandViewDeviceRecordfromListing:
+  Environment: Settings 
+  Roles:
+    - Role: desktopChrome
+      App: "desktop"
+  Actions:
+  # Log in Care Platform as an Provider
+    - Type: case
+      Value: ProvidersCarePlatformLogin
+  # Go to Participants
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform.navigation_participant$      
+#CXTB-171: Access Participant Directory & Filter
```

Comment:
Could you put this on top with the CXTB-170?

Reply:
understood, already fixed.